Adafruit_BluefruitLE_SPI.cpp

```diff
@@ -590,6 +622,7 @@ bool Adafruit_BluefruitLE_SPI::getPacket(sdepMsgResponse_t* p_response)
     spixfer(p_response->payload, p_header->length);
 
     result = true;
+    break; // Uh, no break before? Seriously..how did this work at all?
```

Comment:
Yes, I noticed that later! It is confusing to have a loop just for the purpose of code flow. Instead, the inner portion should be refactored to a separate method. (This is what I did in my branch). It makes the code much more readable and understandable.